static/app/utils/platform.tsx

```diff
@@ -1,4 +1,33 @@
-import {mobile} from 'sentry/data/platformCategories';
+import {
+  backend,
+  desktop,
+  frontend,
+  mobile,
+  PlatformCategory,
+  serverless,
+} from 'sentry/data/platformCategories';
+
+export function platformToCategory(platform: string | undefined): PlatformCategory {
+  if (!platform) {
+    return PlatformCategory.other;
+  }
+  if (([...mobile] as string[]).includes(platform)) {
+    return PlatformCategory.mobile;
+  }
+  if (([...frontend] as string[]).includes(platform)) {
+    return PlatformCategory.frontend;
+  }
+  if (([...backend] as string[]).includes(platform)) {
+    return PlatformCategory.backend;
+  }
+  if (([...serverless] as string[]).includes(platform)) {
+    return PlatformCategory.serverless;
+  }
+  if (([...desktop] as string[]).includes(platform)) {
```

Comment:
If there is a cleaner way to do this, please let me know. I did it like this as it is consistent to the below. I'm also proposing removing `isMobilePlatform` as this function would provide the same functionality.

Reply:
Since this function is essentially converting the string name of an enum value to that value, I'm sure there's an easier way without exhaustively testing every constant. I haven't done enough TypeScript to be sure but take a look at e.g. https://stackoverflow.com/a/42623905 which hopefully has some leads.

Reply:
It's not blocking functionality-wise but it's an easy clean up and it'd be good to address it in this PR for overall code quality

Reply:
I agree! If there was no easy solution then I would be okay to address in the future.